README.md

```diff
@@ -73,7 +73,7 @@ Shaderc into.
 
 On Windows, instead of building from source, you can get the artifacts for
```

Comment:
Please call this experimental. Especially because it's on a BinTray account owned by you rather than by Google.

Also say "The underlying service is not hosted by Google or by GitHub."

Reply:
Done